pkg/jiraloader/jiraloader.go

```diff
@@ -69,32 +80,32 @@ func (jl *JIRALoader) LoadJIRAIncidents() error {
 	}
 
 	for i, issue := range issues.Issues {
-		jiraID, err := strconv.ParseUint(issue.ID, 10, 64)
+		unseenIssues.Delete(issue.Key)
+
+		model, err := issueToDB(&issues.Issues[i])
 		if err != nil {
-			fmt.Printf("parsing error: %+v", err)
+			log.WithError(err).Errorf("couldn't convert jira issue to db model")
 			continue
 		}
-
-		var startTimeP *time.Time
-		if issue.Fields.Created != "" {
-			startTime, err := time.Parse(jiraTimeLayout, issue.Fields.Created)
-			if err != nil {
-				fmt.Printf("parsing error: %+v", err)
-				continue
-			}
-			startTimeP = &startTime
+		if res := jl.dbc.DB.Save(model); res.Error != nil {
+			log.WithError(err).Errorf("couldn't save jira incident to DB")
+			return res.Error
 		}
+	}
 
-		model := &models.JiraIncident{
-			Model: models.Model{
-				ID: uint(jiraID),
-			},
-			Key:            issue.Key,
-			Summary:        issue.Fields.Summary,
-			StartTime:      startTimeP,
-			ResolutionTime: findResolutionTime(&issues.Issues[i]),
+	for _, unseen := range unseenIssues.List() {
+		log.Infof("processing unseen jira incidents (trt-incident label removed?)...")
+		issue, err := queryJiraAPI(unseen)
+		if err != nil {
+			log.WithError(err).Errorf("couldn't query details for %+v", issue)
+			continue
 		}
 
+		model, err := issueToDB(issue)
+		if err != nil {
+			log.WithError(err).Errorf("couldn't convert jira issue to db model")
+			continue
+		}
```

Comment:
I guess we do a query for all issues with label trt-incident over all time and then save them all, maybe something in the future for us to consider only those modified in the last 30 days or something.  Doesn't have to be this PR just walking through it

Reply:
Yea I can limit the jira query, will update